### PR TITLE
Docs/example for GTP linking change

### PR DESCRIPTION
**Pull request by josephpeacock**

### CODE REVIEW

File: docs/Examples.md [hunk 1]
```diff
@@ -266,6 +266,16 @@ Now run the below request multiple times and see the requests being load balance
...
 
+### Global traffic policy
+
+You can add a global traffic policy for the Greeting service to tie all requests to one of the clusters. 
+
+```bash
+kubectl apply -f $ADMIRAL_HOME/yaml/gtp.yaml
+```
+
+Now, when you re-run demo requests, you should see them all flowing to the us-west-2 cluster (cluster 1).
```
Comment (aattuluri) [inline docs/Examples.md:277]:
```suggestion
Now, when you re-run demo requests, you should see them all following to the us-west-2 cluster (cluster 1).
```

File: install/sample/gtp.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  name: gtp-service1
+  namespace: sample
+  labels:
+    identity: greeting
+spec:
+  policy:
+    - dns: default.greeting.global
+      lbtype: FAILOVER
+      target:
+        - region: uswest-2
```
Comment (aattuluri) [inline install/sample/gtp.yaml:13]:
```suggestion
        - region: us-west-2
```

File: install/sample/gtp.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+          weight: 100
+        - region: useast-2
```
Comment (aattuluri) [inline install/sample/gtp.yaml:15]:
```suggestion
        - region: us-east-2
```

File: docs/Architecture.md [hunk 4]
```diff
@@ -123,7 +123,25 @@ Using the Global Traffic policy type will allow for the creation of multiple dns
...
+In this example the service object with the identity=service1 label will have 3 dns names created that map to it.
 - service1.global - pins traffic the local region the traffic originated
 - service1-west.global - sends traffic to the west region and only to east if west in unavailable
 - service1-east.global - sends traffic to the east region and only to west if east in unavailable
+
+
+
+### Global Traffic Policy Linking
+
+A global traffic policy is matched to a deployment if and only if their values for the `workload_sidecar_update` labels match. The default label key for `workload_sidecar_update` is `identity`, meaning that in the default case you can just apply the same identity label that you're already using for your deployments to your global traffic policies.
```
Comment (aattuluri) [inline docs/Architecture.md:135]:
```suggestion
A global traffic policy is matched to a deployment if and only if their values for the `globaltraffic_deployment_label` labels match. The default label key for `globaltraffic_deployment_label` is `identity`, meaning that in the default case you can just apply the same identity label that you're already using for your deployments to your global traffic policies.
```

Comment (josephpeacock) [reply docs/Architecture.md:135]: Good catch

**Pull request merged**